ci/dra_common.sh

```diff
@@ -0,0 +1,33 @@
+function info {
+    echo "INFO: $1"
+}
+
+function error {
+    echo "ERROR: $1"
+}
+
+function save_docker_tarballs {
+    ver=$1
+    arch=$2
+    for image in logstash logstash-oss logstash-ubi8; do
+        # docker save docker.elastic.co/logstash/logstash:${STACK_VERSION} | gzip -c > build/logstash-${STACK_VERSION}-docker-image-x86_64.tar.gz
+        # docker save docker.elastic.co/logstash/logstash-oss:${STACK_VERSION} | gzip -c > build/logstash-oss-${STACK_VERSION}-docker-image-x86_64.tar.gz
+        # docker save docker.elastic.co/logstash/logstash-ubi8:${STACK_VERSION} | gzip -c > build/logstash-ubi8-${STACK_VERSION}-docker-image-x86_64.tar.gz
+        docker save docker.elastic.co/logstash/$image:$ver | gzip -c > build/$image-$ver-docker-image-$arch.tar.gz
+    done
+}
+
+function upload_to_bucket {
+    file=$1
+    gsutil cp $file gs://logstash-ci-artifacts/dra/${STACK_VERSION}/
```

Comment:
This function accesses the Global Variable. The value of this function should be whatever the global variable's latest value is. Not sure if this is a better implementation than doing what's done above.

Reply:
since `STACK_VERSION` is mutated (and not constant-once-set), I would prefer taking a positional argument.